pkg/sql/parser/sql.y

```diff
@@ -4574,10 +4583,106 @@ create_stmt:
 | create_extension_stmt  // EXTEND WITH HELP: CREATE EXTENSION
 | create_external_connection_stmt // EXTEND WITH HELP: CREATE EXTERNAL CONNECTION
 | create_virtual_cluster_stmt     // EXTEND WITH HELP: CREATE VIRTUAL CLUSTER
+| create_logical_replication_stream_stmt     // EXTEND WITH HELP: CREATE LOGICAL REPLICATION STREAM
 | create_schedule_stmt   // help texts in sub-rule
 | create_unsupported     {}
 | CREATE error           // SHOW HELP: CREATE
 
+// %Help: CREATE LOGICAL REPLICATION STREAM - create a new logical replication stream
+// %Category: Experimental
+// %Text:
+// CREATE LOGICAL REPLICATION STREAM 
+//  FROM 'stream_uri' 
+//  ON <TABLE remote_name | TABLES (remote_name, ...) | DATABASE remote_name>
+//  INTO <TABLE remote_name | TABLES (remote_name, ...) | DATABASE remote_name>
+//  [WITH
+//  < MODE= immediate | transactional > |
+//  < CURSOR = start_time > |
+//  < FUNCTION 'udf' FOR TABLE local_name  , ... >
+// ]
+create_logical_replication_stream_stmt:
+  CREATE LOGICAL REPLICATION STREAM FROM d_expr ON logical_replication_resources INTO logical_replication_resources opt_logical_replication_options
+  {
+    /* SKIP DOC */
+    $$.val = &tree.CreateLogicalReplicationStream{
+      PGURL: $6.expr(),
+      On: $8.logicalReplicationResources(),
+      Into: $10.logicalReplicationResources(),
+      Options: *$11.logicalReplicationOptions(),
+    }
+  }
+| CREATE LOGICAL REPLICATION STREAM error // SHOW HELP: CREATE LOGICAL REPLICATION STREAM
+
+logical_replication_resources:
+  TABLE db_object_name
+  {
+    $$.val = tree.LogicalReplicationResources{
+      Tables: tree.TablePatterns{
+        $2.unresolvedObjectName().ToUnresolvedName(),
+      },
+    }
+  }
+| logical_replication_resources ',' db_object_name
```

Comment:
I think the doc had this as `TABLES (a, b, c)`, using the plural to separate it from the single table version and wrapping the list in parens to make it unambiguous where it starts and stops.

We regretted doing something like the version in the diff now in backup where we allowed a multi-item bare names separated by commas without wrapping in parens in a position where we also sometimes have a qualifier on a name (table, database, tenant) since it meant non-reserved keywords are ambiguous.

Reply:
Ah yes. I've updated the parser to reflect this now.